src/hash.cr

```diff
@@ -1611,42 +1611,45 @@ class Hash(K, V)
 
   # Returns a new hash with all keys converted using the block operation.
   # The block can change a type of keys.
+  # The block yields the key and value.
   #
   # ```
   # hash = {:a => 1, :b => 2, :c => 3}
-  # hash.transform_keys { |key| key.to_s } # => {"a" => 1, "b" => 2, "c" => 3}
+  # hash.transform_keys { |key, _value| key.to_s } # => {"a" => 1, "b" => 2, "c" => 3}
```

Comment:
This addition isn't really necessary. Unused block parameters can simply be omitted, so I don't think it's useful to write it it with an underscore prefix.
```suggestion
  # hash.transform_keys { |key| key.to_s } # => {"a" => 1, "b" => 2, "c" => 3}
```

Ditto below.